engine-plugins/model-plugins/src/main/scala/org/trustedanalytics/atk/engine/model/plugins/timeseries/MAXJsonProtocol.scala

```diff
@@ -0,0 +1,82 @@
+/**
+ *  Copyright (c) 2015 Intel Corporation 
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.trustedanalytics.atk.engine.model.plugins.timeseries
+
+import com.cloudera.sparkts.models.{ ARIMAXModel => MAXModel }
```

Comment:
Since we are using the same sparkts model for both ARIMAX and MAX, maybe we don't need to have this MAXJsonProtocol?  Just use the import the ARIMAXJsonProtocol - can you try that?